libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.spec.ts

```diff
@@ -203,4 +205,106 @@ describe(WattNavListModule.name, () => {
       expect(link).not.toHaveAttribute('target');
     });
   });
+
+  describe('Supports expandable nav list', () => {
+    it('can set a title on an expandable nav list', async () => {
+      const title = 'Expandable menu';
+
+      const view = await render(
+        `
+        <watt-nav-list [title]="'${title}'" [expandable]="true">
+          <watt-nav-list-item link="/sub-page">Sub page</watt-nav-list-item>
+        </watt-nav-list>
+      `,
+        {
+          imports: [WattNavListModule],
+        }
+      );
+
+      expect(view.queryByText(title)).not.toBeNull();
```

Comment:
Its nice to know that its not null, for sure, but can i suggest that you make an expect that checks that the title is actually set to what we input? :)

Reply:
Isn't that implicitly true by using `view.queryByText(title)`? If the title is not found in the DOM, the query will not find it.

Reply:
https://github.com/Energinet-DataHub/greenforce-frontend/pull/831/files#r959194250